server/src/main/resources/db/migration/V1_1__Init.sql

```diff
@@ -6,6 +6,9 @@ CREATE TABLE IF NOT EXISTS scope_permissions (
     created TIMESTAMPTZ NOT NULL
  );
 
+CREATE INDEX IF NOT EXISTS scope_permissions_scope_address ON scope_permissions (scope_address);
```

Comment:
Forgot to include these indexes in the flyway pr, shouldn't matter to just add them here as this hasn't been run in any real environment

Reply:
I was actually adding this in my "add grant ID" PR so this is perfect, thanks